### ISSUE TITLE
Shutdown Button - Dashboard

### ISSUE THREAD

**Issue by ayz1e** [CatalysmsServerManager/7-days-to-die-server-manager#275]
Restart button currently only issues a shutdown command.

I would like the restart button to issue a kickall command, followed by the shutdown. Players not connecting from the server prior to a shutdown can result in errors.

**Comment by niekcandaele** [comment]
This is the controller that should be updated for this:
https://github.com/CatalysmsServerManager/7-days-to-die-server-manager/blob/master/api/controllers/SdtdServer/restart-server.js

**Comment by EmberCS15** [comment]
Hey I could not capture the exact requirements from the conversation - 

Is it - "restart button to issue a kickall command, followed by the shutdown. "
Or is it "produce a cleaner message for online players" ?
Or are both interlinked ?

**Comment by ayz1e** [comment]
Both are interlinked. By adding the kickall command with the message "The
server is restarting." it will create a cleaner message for players.

**Comment by niekcandaele** [comment]
Yep, so what happens now is when the button gets pressed, the only command executed is the `shutdown` command. This should be changed to 2 commands:

`kickall "Server will be restarted, please try again in a few minutes"`
Followed by
`shutdown`

**Comment by niekcandaele** [comment]
Adding to this, the controller currently still uses the old machinepack lib. To make this a lot simpler, should be refactored to use the new wrapper lib ( https://github.com/CatalysmsServerManager/7-Days-to-Die-API-wrapper ). That dep is already in there, just need to switch them out

**Comment by EmberCS15** [comment]
Thanks for the clarification, I did some investigation for this and these are the points that I wanted to verify - 

1. Instead of using machinepack lib we will plug in the new wrapper lib by simply changing the library import in [require](https://github.com/CatalysmsServerManager/7-days-to-die-server-manager/blob/master/api/controllers/SdtdServer/restart-server.js#L1). This, hopefully does not have a side effect

2. In [this](https://github.com/CatalysmsServerManager/7-days-to-die-server-manager/blob/6de3006f6b0741e540c1f0e0e02c98a32f7c7279/api/controllers/SdtdServer/restart-server.js#L55) place we will issue 2 separate commands of `kickall with new message` followed by `shutdown with appr. message`

**Comment by niekcandaele** [comment]
1. It's a bit more work than just changing the require :) do a search for `sails.helpers.sdtdApi`to see how to use it, it's pretty straight forward
2. Correct!
